Gemfile.lock

```diff
@@ -37,6 +37,8 @@ GEM
       minitest (~> 5.1)
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
+    addressable (2.5.1)
```

Comment:
@SamSaffron  I know you hate addressable for its memory footprint but we'll only be installing this in the test environment. `FakeWeb` hasn't been updated for a couple of years and doesn't work with Ruby 2.4.1 at all. I've had to switch to `WebMock` which introduces this dependency.